packages/plugin-trpc/cli/tests/lib/generate.spec.ts

```diff
@@ -0,0 +1,301 @@
+import { test } from 'uvu';
```

Comment:
Started with tests also for the generate file. In case we refactor it, we maybe need to change the structure of the test files as well